File: Ejercicios POO/Javascript/POO/usuarios.js
// Written By Ismael Heredia in the year 2016

function Usuario(id_usuario,nombre,password,tipo,fecha_registro){
	this.id_usuario = id_usuario;
	this.nombre = nombre;
	this.password = password;
	this.tipo = tipo;
	this.fecha_registro = fecha_registro;
}

Usuario.prototype.getId_usuario=function(){
	return this.id_usuario;
};
Usuario.prototype.setId_usuario=function(id_usuario){
	this.id_usuario = id_usuario;
};

Usuario.prototype.getNombre=function(){
	return this.nombre;
};
Usuario.prototype.setNombre=function(nombre){
	this.nombre = nombre;
};

Usuario.prototype.getPassword=function(){
	return this.password;
};
Usuario.prototype.setPassword=function(password){
	this.password = password;
};

Usuario.prototype.getTipo=function(){
	return this.tipo;
};
Usuario.prototype.setTipo=function(tipo){
	this.tipo = tipo;
};

Usuario.prototype.getFecha_registro=function(){
	return this.fecha_registro;
};
Usuario.prototype.setFecha_registro=function(fecha_registro){
	this.fecha_registro = fecha_registro;
};

Usuario.prototype.toString=function(){
	return "[+] ID Usuario : "+this.id_usuario+"\n"+"[+] Nombre : "+this.nombre+"\n"+"[+] Password : "+this.password
			+"\n"+"[+] Tipo : "+this.tipo+"\n"+"[+] Fecha registro : "+this.fecha_registro;
};

File: CRUD/Python/Sistema con SQLServer/Usuarios.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
# Written By Ismael Heredia in the year 2017

class Usuario(object):

   def __init__(self, id="", nombre="", clave="",id_tipo="",fecha_registro=""):
       self.__id = id
       self.__nombre = nombre
       self.__clave = clave
       self.__id_tipo = id_tipo
       self.__fecha_registro = fecha_registro
              
   def getId(self):
       return self.__id

   def setId(self, id = ""):
       self.__id = id

   def getNombre(self):
       return self.__nombre

   def setNombre(self, nombre = ""):
       self.__nombre = nombre

   def getClave(self):
       return self.__clave

   def setClave(self, clave = ""):
       self.__clave = clave
    
   def getId_tipo(self):
       return self.__id_tipo

   def setId_tipo(self, id_tipo = ""):
       self.__id_tipo = id_tipo
       
   def getFecha_registro(self):
       return self.__fecha_registro

   def setFecha_registro(self, fecha_registro = ""):
       self.__fecha_registro = fecha_registro
       
   def toString(self):
     return "Usuario{id=%d, nombre=%s, clave=%s, id_tipo=%d, fecha_registro=%s}" % (self.__id,self.__nombre,self.__clave,self.__id_tipo,self.__fecha_registro)
   
   def destroy(self):
       self.__id = None
       self.__nombre = None
       self.__clave = None
       self.__tipo = None
       self.__fecha_registro = None
